src/components/locationForm/LocationForm.jsx

```diff
@@ -43,14 +56,23 @@ const LocationForm = ({ id }) => {
     setValue(e.target.name, value);
   };
 
+  const onReset = () => {
+    history.push(appRoutes.HOME);
+  };
+
   useEffect(() => {
     Object.keys(initalValues).map(name => register({ name }));
     ['rapport_document', 'design_document'].map(name => register({ name }));
-  }, [register]);
+  }, [register, id]);
 
   return (
     <>
-      <form onSubmit={handleSubmit(onSubmit)} action="" noValidate>
+      <form
+        onSubmit={handleSubmit(onSubmit)}
+        action=""
+        noValidate
+        onReset={onReset}
```

Comment:
Doesn't seem to be supported by IE: https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers/onreset

Reply:
thanks, changed.